### DISCUSSION TITLE
PF2e Missing Content

### DISCUSSION THREAD

**Discussion by mcDevnagh** [ebullient/ttrpg-convert-cli#343] [Q&A]
Is there anyway to have ttrpg-convert-cli create markdown for the following big ticket items from [https://pf2etools.com/](https://pf2etools.com/)?
Big ticket meaning they get their own link at https://pf2etools.com/
- [ancestries](https://pf2etools.com/ancestries.html)
- [bestiary](https://pf2etools.com/bestiary.html)
- [classes](https://pf2etools.com/classes.html)

It looks like none is included here: https://github.com/ebullient/ttrpg-convert-cli/blob/51945c2a2657490c7a5a1478500c6da9a2be0c95/src/main/resources/convertData.json#L465

**Comment by ebullient** [comment]
It has been on the list for awhile. Viability of Pf2eTools as a source is a question. There are some other options on the table, see https://discord.gg/Zpmr37Uv for folks to talk to about it. ;)

**Reply by mcDevnagh** [reply]
Link expired :(

**Reply by ebullient** [reply]
https://discord.com/channels/925322340995313686/1170261597969731665

Obsidian TTRPG Community Discord

**Comment by mcDevnagh** [comment]
I missed these issues! My bad:
- #226 
- #227 
- #228

**Reply by ebullient** [reply]
@miscoined has been working on this.. though we haven't properly linked to some of those old items that may be closable now..

**Reply by miscoined** [reply]
Bestiary progress is WIP but not complete. The majority of content is there but it's missing a few things like abilities and spells.
